presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/json/format/SecondsSinceEpochFormatter.java

```diff
@@ -27,6 +27,8 @@
 {
     public static boolean isSupportedType(Type type)
     {
+        // seconds-since-epoch cannot encode a timezone hence writing TIMESTAMP WITH TIME ZONE
+        // is not supported to avoid losing the irrecoverable timezone information after write.
```

Comment:
same here:

i think it would be OK to write sans zone
https://github.com/prestosql/presto/blob/fbd4ca7f6ca1326a077fd06cd33b687cbfe17cf2/presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java#L612-L628

Reply:
Same here. TODO would be nice.